feature-flags.md

```diff
@@ -0,0 +1,76 @@
+# Lightning feature flags: from Zero to Hero
+
+This article describes optional lightning features that nodes can turn on and off.
+Nodes actively tell the whole network about the features they support, to help peers looking for specific features connect to them.
+
+## `data_loss_protect`
+
+This flag allows a node to detect upon channel reestablishment that it has lost state (i.e. thinks a state is 'current' while it has been superseded and maybe even revoked). This works because the commitment point (that is added with this flag) can be verified to be authenticly derived from the secret, so the 'forgetful' node must have sent it to them before it lost its commitment point counter state.
```

Comment:
nit:

```suggestion
This flag allows a node to detect upon channel reestablishment that it has lost state (i.e. thinks a state is 'current' while it has been superseded and maybe even revoked). This works because the commitment point (that is added with this flag) can be verified to be authentically derived from the secret, so the 'forgetful' node must have sent it to them before it lost its commitment point counter state.
```